src/classes/HH_Container_LCTRL.cls

```diff
@@ -327,7 +329,6 @@ public with sharing class HH_Container_LCTRL {
     * @description deletes the Contacts in the database
     * @param listCon The list of Contacts to delete
     * @return void
-    */
```

Comment:
can't we just delete the method, or will it for sure be returning in the near future?